src/bootstrap/lib.rs

```diff
@@ -132,7 +132,7 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &'static str, Option<&[&'static str]>)]
     (Some(Mode::Std), "backtrace_in_libstd", None),
     /* Extra values not defined in the built-in targets yet, but used in std */
     (Some(Mode::Std), "target_env", Some(&["libnx"])),
-    // (Some(Mode::Std), "target_os", Some(&[])),
+    (Some(Mode::Std), "target_os", Some(&["teeos"])),
```

Comment:
```suggestion
    (Some(Mode::Std), "target_os", Some(&["teeos"])), // #[cfg(bootstrap)]
```
This tells the release team to remove this again after the bootstrap compiler supports teeos.

Reply:
accepted